packages/react-components/react-dialog/src/stories/DialogTitle/DialogTitleCustomAction.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
```

Comment:
Since the `DialogTrigger` can be used to close. Should we add another story to show how to create a completely custom action that closes the dialog ?